README.adoc

```diff
@@ -198,6 +198,68 @@ include::{common-includes}/devmode-build.adoc[]
 Check out the service that you created at the
 http://localhost:9080/LibertyProject/System/properties[^] URL. 
 
+// =================================================================================================
+// Exception Handling
+// =================================================================================================
+
+== Exception Handling
+
+If a resource doesn't exist or the service experiences an exception, the client will receive either an empty response or an HTML error page respectively.
+A client should not receive either of these.
+Clients should receive JSON responses. 
```

Comment:
I would rephrase this section.  I'm not sure that the client will always receive an empty response or an HTML error page.  And ideally, the client would receive the error response in the same MediaType (MIME type) that they would receive a normal response in (i.e. it could be text, xml, etc.).

Maybe something like this:

Most application developers will want to ensure that when an exception occurs in the application (such as the client requesting a resource that does not exist or posting an invalid resource, etc.) that it is handled consistently and in a way that allows the client to fully understand the failure (i.e. bad input, server outage, etc.). This can be accomplished using the exception handling mechanisms in JAX-RS such as sending a `Response` via a `WebApplicationException` (or subclass) or by using `ExceptionMapper` providers.